src/clj/rems/css/styles.clj

```diff
@@ -417,9 +417,14 @@
    [:.alert-dark {:color (util/get-theme-attribute :alert-dark-color)
                   :background-color (util/get-theme-attribute :alert-dark-bgcolor)
                   :border-color (util/get-theme-attribute :alert-dark-bordercolor :alert-dark-color)}]
+   [:#navbar
```

Comment:
The distinction between `#navbar` and `.navbar` seems arbitrary. Maybe rename `#navbar` to `.navbar-wrapper` or something more descriptive.